...the program crashes and prints the exception
